src/EventStore.ClusterNode/ClusterVNodeHostedService.cs

```diff
@@ -36,41 +36,64 @@ internal class ClusterVNodeHostedService : EventStoreHostedService<ClusterNodeOp
 		protected override IEnumerable<OptionSource>
 			MutateEffectiveOptions(IEnumerable<OptionSource> effectiveOptions) {
 			var developmentOption = effectiveOptions.Single(x => x.Name == nameof(ClusterNodeOptions.Dev));
+			var insecureOption = effectiveOptions.Single(x => x.Name == nameof(ClusterNodeOptions.Insecure));
 			bool.TryParse(developmentOption.Value.ToString(), out bool developmentMode);
+			bool.TryParse(insecureOption.Value.ToString(), out bool insecureMode);
 			return effectiveOptions.Select(x => {
 				if (x.Name == nameof(ClusterNodeOptions.MemDb)
 				    && x.Source == "<DEFAULT>"
 				    && developmentMode) {
 					x.Value = true;
-					x.Source = "Set by 'Development Mode' mode";
+					x.Source = "Set by 'Development' mode";
+				}
+
+				if (x.Name == nameof(ClusterNodeOptions.DisableInternalTcpTls)
+				    && x.Source == "<DEFAULT>"
+				    && insecureMode) {
+					x.Value = true;
+					x.Source =  "Set by 'Insecure' mode";
+				}
+
+				if (x.Name == nameof(ClusterNodeOptions.DisableExternalTcpTls)
+				    && x.Source == "<DEFAULT>"
+				    && insecureMode) {
+					x.Value = true;
+					x.Source =  "Set by 'Insecure' mode";
+				}
+
+				if (x.Name == nameof(ClusterNodeOptions.DisableHttps)
+				    && x.Source == "<DEFAULT>"
+				    && insecureMode) {
+					x.Value = true;
+					x.Source =  "Set by 'Insecure' mode";
 				}
 
 				if (x.Name == nameof(ClusterNodeOptions.CertificateFile)
 				    && x.Source == "<DEFAULT>"
-				    && developmentMode) {
+				    && developmentMode && !insecureMode) {
```

Comment:
Could this lead to a situation where a user expects dev certificates to be set up, but they aren't because I'm also running in insecure mode? e.g:
```
EventStore.ClusterNode.exe --dev --insecure --disable-external-tcp-tls=false
```
This would fail further down because no certs were specified but I have a secured interface.

Reply:
@hayley-jean yup, we can skip these `insecureMode` checks and just set the dev certificates so that the --dev option is more consistent. If --insecure is specified, the certificates would not be used anyway.

Reply:
This will be addressed when tackling: https://github.com/EventStore/home/issues/213